### ISSUE TITLE
Merge TypeOfThing sheet and MeasureCategory sheet?

### ISSUE THREAD

**Issue by anirudh4792** [ChildMindInstitute/mhdb#60]
I have classified **Things** into 
claim made by project to improve or reduce a specific condition, state or symptom; 
addresses mental/psychological state; 
addresses physical/bodily state; 
Concepts addressed by the project

and **Measure Category** into 
physical property; 
physiological property; 
speech (eg jitter, shimmer, amplitude etc. come under speech)
movement

Do you feel these could be merged (I feel there are overlaps).

**Comment by binarybottle** [comment]
Is this not covered in greater depth by other issues?  If so, please close.

**Comment by anirudh4792** [comment]
Update - After adding more measures that map to signs, the measure category is useful to link measures such as frequency of tremors and respiration to categories like movement. However, it does not necessarily represent inter-measure relatedness. For example, liquid has been added to measure pupil dilation, and liquid is related to tears (used as a measure for lacrimation). 

For language disorders, I have added number, quantity and count as measures for spelling mistakes, reading accuracy etc.
